src/Nvtrust.proto

```
@@ -0,0 +1,12 @@
+syntax = "proto3";
```

Buf (1.47.2)

Files must have a package defined.

(PACKAGE_DEFINED)